Applications/IK/test/constraintTest_ikOnly.osim

```diff
@@ -5941,7 +5941,7 @@
 						<groups/>
 					</WrapObjectSet>
 				</Body>
-				<Body name="toes_l">
+				<Body name="toes_left">
 					<mass>       0.09750000 </mass>
```

Comment:
@aseth1 Indeed, the model has a Body and Joint with the same name so findComponent was returning the wrong type and throwing an exception that it's not finding the PhysicalFrame/Body. If the search is done by name then we need to make the name unique and hopefully enforce the uniqueness downstream.